README.md

```diff
@@ -25,3 +25,29 @@ npm run build
 ```
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+## Internationalization (i18n)
+
+You are welcome to submit translations of the wiki pages. So far we only have a small portion of the wiki translated to French. We currently only have capacity to review the following languages:
+
+- French
+- German
+- Swedish
+- Turkish
+
+If your translation is a mere Google translation without any correction, the Pull Request will likely not be accepted.
+
+**Before you begin translating, please consider your submission as *commitment* to maintain the pages in sync with any changes happening in the english version**
+
+### Enable a new language for translation
+
+1. Create a folder for the documents inside the `i18n` folder. For example german: `mkdir -p i18n/<two-letter-international-code>/docusaurus-plugin-content-docs/current` (*note that you will need replace `<two-letter-international-code>` with the actual two-letter international code*).
+2. Copy all the content from `/docs` into the new folder: `cp -r docs i18n/<two-letter-international-code>/docusaurus-plugin-content-docs/current/` (*again, note that you will need replace `<two-letter-international-code>` with the actual two-letter international code*).
+3. Remove all the `_category_.json` files from the subfolders inside `i18n/<two-letter-international-code>/docusaurus-plugin-content-docs/current`.
+4. Build the translation files for the navbar and the footer: `npm run write-translations -- --locale <two-letter-international-code>`.
+
+Now you can start translating any and all file in `i18n/<two-letter-international-code>/docusaurus-plugin-content-docs/current`
```

Comment:
```suggestion
Now you can start translating all files in `i18n/<two-letter-international-code>/docusaurus-plugin-content-docs/current`
```

Reply:
all AND any because you don't have to translate all of them if you only have caoacity for one